client/src/services/AuthService.js

```diff
@@ -22,6 +22,14 @@ export default {
             .then((res) => res.json())
             .then((data) => data);
     },
+    withCookie: () => {
+        return fetch("/auth/user", {
+            method: "get",
+            credentials: 'include'
+        })
+            .then((res) => res.json())
+            .then((data) => data);
+    }
```

Comment:
We can just move the contents of this entire file into userContext, since userContext is basically serving as our auth service

Reply:
Removed this file as all info now lives in the UserContextProvider